src/karapace/api/oidc/middleware.py

```diff
@@ -35,6 +35,8 @@ def __init__(self, app: FastAPI, config: Config) -> None:
                 raise ValueError(
                     "OIDC config error: 'issuer' and 'audience' must be set if 'jwks_endpoint_url' is provided."
                 )
+            log.info("OIDC middleware initialized — JWKS URL configured, bearer token validation enabled.")
+            log.debug("OIDC Params jwks_url=%s issuer=%s audience=%s", self.jwks_url, self.issuer, self.audience)
```

Comment:
Maybe revert this log line or unless none of those details would reveal sensitive information 👍

Reply:
It is indeed helpful, I can change it to info. Good point.